katsdptelstate/telescope_state.py

```diff
@@ -659,9 +659,24 @@ def backend(self):
         return self._backend
 
     def load_from_file(self, file):
-        """Load keys from a Redis-compatible RDB file (as filename or object).
+        """Load keys from a Redis-compatible RDB snapshot file.
 
-        Will raise ImportError if the rdbtools package is not installed.
+        Parameters
+        ----------
+        file : str or file object
+            Filename or file object representing RDB file
+
+        Returns
+        -------
+        keys_loaded : int
+            Number of keys loaded from RDB file into telstate
+
+        Raises
+        ------
+        ImportError
+            If the rdbtools package is not installed
+        RdbParseError
+            If the file could not be parsed (truncated / malformed / not RDB)
```

Comment:
Explicitly mention somewhere that errors raised while opening the file and errors raised by the backend can also occur.

Reply:
Done.